### ISSUE TITLE
Game Editor: Changing the name of the red/green/blue/black buttons

### ISSUE THREAD

**Issue by IvonArroyo** [wearable-learning-cloud-platform/wlcp-issues#15]
Many students expressed they wanted to change the name of the buttons. 
This should not be hard to do and is a very useful enhancement.

**Comment by IvonArroyo** [comment]
There is a bug with the colored buttons. Take a look at this scenario. I was trying to get the RED button say "get 1 point" and the GREEN button to say "get 2 points". However, the names don't repeat across transitions that are at the same level. Also, when debugging it chooses one of the two labels to display, but not both labels.

<img width="1598" alt="Screen Shot 2022-02-15 at 3 30 22 PM" src="https://user-images.githubusercontent.com/5631787/154712532-bd861851-dda5-4059-bb9a-5bf7e585d4dc.png">
:
<img width="799" alt="Screen Shot 2022-02-18 at 10 31 35 AM" src="https://user-images.githubusercontent.com/5631787/154713235-aba30726-c2f3-4894-9315-13a2286e2652.png">
<img width="823" alt="Screen Shot 2022-02-18 at 10 31 43 AM" src="https://user-images.githubusercontent.com/5631787/154713248-0a9badeb-994a-4324-a4f0-069003978da7.png">

**Comment by mmicciolo** [comment]
@IvonArroyo @franciscastro @sai-educ @geseiche @apoh3

Bug found. Moving back to development.

Thanks!

**Comment by mmicciolo** [comment]
Hello @IvonArroyo @franciscastro @sai-educ @geseiche @apoh3 

I have fixed this bug and deployed to dev.

Since a test case was never written I will copy the mail I sent to the ERSP team.

There is a bug in the button naming.

This is how it should work. Example below.

![image](https://user-images.githubusercontent.com/23061418/156237057-7f2e2984-027b-4fb4-b45b-299cafbbe1fc.png)

Opening up the transition of the left before state A.

![image](https://user-images.githubusercontent.com/23061418/156237076-91924949-d71b-48db-a810-a38c6e834a90.png)

Opening up the transition on the right before state B.

![image](https://user-images.githubusercontent.com/23061418/156237090-7c4431f4-d3db-4379-8f08-7adfe9ceaafa.png)

As you can see since the first checkbox (GO TO A) is being used in the first transition (left one), when you open up the second transition (right one) the first checkbox should be greyed out (as its already selected in the other transition) and the label should be the same as the label you defined in the first transition (left one).

Right now there is a bug where the labels do not line up and in the second transition (right one) the label for the first checkbox gets stuck at Red Button.

The issue you are facing with some buttons showing null for the label is something different. That has to do with the fact that new.wearablelearning.org is running a older version of the platform where we don’t have button label editing.

Therefore, when I exported/imported your games from new to dev there was no label data so they defaulted to null. I will run a small script to change all of your button labels to the default, red, green, blue black. You’ll need to go back in and rename them if you have already started changing them.

**Comment by geseiche** [comment]
**Test Passed**

I set up the testing scenario listed above. I observed correct propagation of new label names between different transitions. I tested the situation in the debugger and in the game player. Both place showed the correct labels. 

_Note:_ I did not observe the bug of labels not lining up that is mentioned in the test script

_Potential Implementation Change:_ Editors may appreciate being able to name all the buttons at once instead of only when they are selected in the transition. (This would mean NOT greying out the labels when they are not selected)
